WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PersonsController < ApplicationController
+  def index
+    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
+                 'Country' => Country.all.map { |country| [country.name, country.id] } }
+
+    params[:region] ||= "all"
+
+    respond_to do |format|
+      format.html
+      format.js do
+        persons = Person.joins("JOIN Countries ON countryId = Countries.id")
```

Comment:
Feel free to ignore this, but when I see `persons` I expect a collection of `Person` objects. I'd call this one `query`.